config/locales/en.yml

```diff
@@ -123,6 +135,16 @@ en:
             content_html: If you don’t know the date you were cautioned, you can get this information for free by asking the police for a <a href='https://www.gov.uk/copy-of-police-records'>subject access request (SAR)</a>.
           date_legend: Enter the date
           date_hint: For example, 29 1 2018
+      conditional_end_date:
```

Comment:
Can we add the copy as in the prototype/design? I know it might eventually change but it makes it easier to review and to test locally.